src/styles.css

```diff
@@ -31,6 +31,10 @@ h1, h1, h3, h4, h5, h6 {
   text-align: center;
 }
 
+.Modal {
+  background-color: rgba(0, 0, 0, 0.75);
```

Comment:
I honestly wasn't sure how to make the background be the primary color of the user's current theme with the transparency effect; we may want to revisit this later if we want the background to have the current theme color when the overlay opens and still have the transparency effect be active.

Reply:
This fine. Realistically, we will have to write CSS in these specific scenarios. But for 99% of cases, we should be using tailwind.

Reply:
> I honestly wasn't sure how to make the background be the primary color of the user's current theme with the transparency effect; we may want to revisit this later if we want the background to have the current theme color when the overlay opens and still have the transparency effect be active.

Maybe this has already been answered but "var(--primary-color)" should work, if I'm not mistaken.

Reply:
> Maybe this has already been answered but "var(--primary-color)" should work, if I'm not mistaken.

Thanks for the suggestion. How do I combine it with the transparent background effect? I wasn't sure how to integrate `var(--primary-color)` into `background-color: rgba(0, 0, 0, 0.75);`.